runtime/binding-grpc/src/main/java/io/aklivity/zilla/runtime/binding/grpc/internal/config/GrpcBindingConfig.java

```diff
@@ -58,26 +71,41 @@ public final class GrpcBindingConfig
     private static final byte[] BIN_SUFFIX = "-bin".getBytes();
     private final HttpGrpcHeaderHelper helper;
 
+    private final Long2ObjectHashMap<CatalogHandler> handlersById;
+    private final Int2ObjectHashMap<GrpcProtobufConfig> protobufsBySchemaId;
+    private final List<CatalogedConfig> catalogs;
+    private final Int2IntHashMap schemaIds;
+
+    public final List<GrpcRouteConfig> routes;
     public final long id;
     public final String name;
     public final KindConfig kind;
     public final GrpcOptionsConfig options;
```

Comment:
All the (non-static) public final fields should be before the (non-static) private final fields.